contributions/contribution.yaml

```diff
@@ -483,6 +483,30 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /contributions/version:
+    get:
+      summary: Get version of logging building block
+      tags:
+        - Contributions
+      responses:
+        200:
+          description: The version of logging building block.
+        400:
+          description: Bad request
+        500:
+          description: Internal server error
+  /contributions/buildingblocks:
```

Comment:
The Dockerfile also needs to be updated to copy the JSON file. Thanks.

Reply:
could you try again after pulling the changes

Reply:
I tried after getting the latest. These are the comments that I'm using. I think it could be the path inside the docker image.
 
```
docker build --pull -t rokwire/contributions-building-block:latest . && docker run -p 5000:5000 --env MONGO_CONTRIBUTION_URL=mongodb://host.docker.internal:27017 --env-file contributions/.env  --name contributions --rm rokwire/contributions-building-block:latest 
```

Reply:
The getCwd path was `/usr/src/app/contributions/api`. I have made a few final suggestions. I think we can merge this PR after that. Thanks.

Reply:
I think that I implemented your suggestions

Reply:
Thank you!